modules/Microsoft.ContainerService/managedClusters/.test/flux/deploy.test.bicep

```diff
@@ -0,0 +1,105 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.containerservice.managedclusters-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'csmmf2'
```

Comment:
Why the 2 additional tests? Can we include all parameters to be validated to the existing common test?

Reply:
The two tests was there for 2 reasons. (1) Test the minimal config and a more normal config (common). So that (2) the two variants are shown in the readme.

Reply:
Personally, I'd vote for keeping deployment tests at a minimum, unless not feasible due to parameter conflicts. Especially for cross referenced resources, the relative min is already validated as part of their module tests.

Reply:
minFlux test is removed

Reply:
I'd still suggest to merge the flux test with the common for the parent resource, unless it creates conflicts. The purpose of the common test is to validate all possible input parameters, including child and cross referenced modules, not to provide an example of the most common scenario, although I agree the name is confusing :)

Reply:
Done